api/src/main/java/jakarta/batch/runtime/BatchRuntime.java

```diff
@@ -43,28 +43,33 @@
 	* @return JobOperator instance.
 	*/
 	public static JobOperator getJobOperator() {
+		JobOperator operator = null;
+		if (System.getSecurityManager() == null) {
+			for (JobOperator op : ServiceLoader.load(JobOperator.class)) {
+				operator = op;
+				break;
+			}
+		} else {
+			operator = AccessController.doPrivileged(new PrivilegedAction<JobOperator>() {
+				public JobOperator run() {
 
+					ServiceLoader<JobOperator> loader = ServiceLoader.load(JobOperator.class);
+					JobOperator returnVal = null;
+					for (JobOperator provider : loader) {
+						if (provider != null) {
+							if (logger.isLoggable(Level.FINE)) {
+								logger.fine("Loaded BatchContainerServiceProvider with className = " + provider.getClass().getCanonicalName());
```

Comment:
Seems like  "JobOperator" would be more helpful than "BatchContainerServiceProvider" .   (I know you didn't change this code).

Reply:
changed to "JobOperator" in log message.